internal/kafka/internal/services/kafka_tls_certificate_management/certificate_revocation_reason.go

```diff
@@ -0,0 +1,46 @@
+package kafka_tls_certificate_management
```

Comment:
Having a package with underscores is quite uncommon and not idiomatic.
Can we rename the package name? for example to certmgmt or kafkacertmgmt. This package is a subdirectory of  a `kafka` directory

Reply:
I am okay to renaming it but 

> Can we rename the package name? for example to certmgmt or tlscertmgmt, or kafkatlscertmgmt. This package is a subdirectory of a kafka directory

I'll go with `kafkatlscertmgmt` though I am not that much of a big fan of acronyms / shortening names.

Reply:
Thinking about it. It might be better to have it with underscore. If it makes it hard to read or not evident then I think it is acceptable to keep it long / with underscores.

This is further reinforced by Go authors themselves https://go.dev/blog/package-names :
`On the other hand, if abbreviating a package name makes it ambiguous or unclear, don’t do it.`

Reply:
I do not know what happened and only seeing this now after changing to `kafkatlscertmgmt`.
I'll keep the suggested package name for now. We can always rename if it become confusing. 
Thanks.